### DISCUSSION TITLE
Install plugins

### DISCUSSION THREAD

**Discussion by epick** [netbox-community/netbox#10577] [Q&A] [answered]
### NetBox version

v3.3.4

### Python version

3.8

### Steps to Reproduce

1. $ source /opt/netbox/venv/bin/activate
(venv) $ pip3 install nextbox-ui-plugin
`(venv) user@netbox:/opt/netbox/nextbox-ui-plugin$ sudo pip3 install .
Processing /opt/netbox/nextbox-ui-plugin
Building wheels for collected packages: nextbox-ui-plugin
  Building wheel for nextbox-ui-plugin (setup.py) ... done
  Created wheel for nextbox-ui-plugin: filename=nextbox_ui_plugin-0.9.2-py3-none-any.whl size=1256040 sha256=a133ebbb7da77203eb87e1cc954ddd21de9a6aab993442540e8a6ae3dd9ccf79
  Stored in directory: /root/.cache/pip/wheels/3b/c3/e0/66ca93287095a9431049a1c8e9f662c00351fa3ed1051ffd3d
Successfully built nextbox-ui-plugin`

2. echo nextbox-ui-plugin >> local_requirements.txt

3. PLUGINS = ['nextbox_ui_plugin'] in a global Netbox configuration.py configuration file
4. sudo systemctl restart netbox netbox-rq

### Expected Behavior

See on the menu the plugin option

### Observed Behavior

502 Bad Gateway

If I remove the name of the plugin in the configuration.py system work again

**Comment by bogdancordos** [comment, marked as answer]
You might need to run `python3 manage.py migrate` before restarting nebox.

**Reply by epick** [reply]
Thanks!!
